birdhouse/deployment/deploy.sh

```diff
@@ -178,8 +178,11 @@ set -x
 # restart everything, only changed containers will be destroyed and recreated
 ./pavics-compose.sh up -d
 
-# immediately cache the new notebook image for faster startup by JupyterHub
-docker pull $DOCKER_NOTEBOOK_IMAGE
+# immediately cache the new notebook images for faster startup by JupyterHub
+for IMAGE in ${DOCKER_NOTEBOOK_IMAGES}
+do
+  docker pull $IMAGE
```

Comment:
Ah true, forgot docker pull can not accept multiple images.

Reply:
Oh wait, this same docker pull was already in the `pavics-compose.sh` script which is called earlier already.  I think we can simply delete this.